pkg/orchestration/wiring/testdata/internaldns.multipleresources.bundle.yaml

```diff
@@ -57,6 +57,27 @@ spec:
                 resourceName: c1
                 stateName: state1
             spec:
+              affinity:
+                podAntiAffinity:
+                  preferredDuringSchedulingIgnoredDuringExecution:
+                  - podAffinityTerm:
+                      labelSelector:
+                        matchExpressions:
+                        - key: app
```

Comment:
Discussed offline - fixing to refer to one kubecompute by matching a resource label instead of a service label

Reply:
Updated to use the resourceName as the key.